docs/guides/cloud.rst

```diff
@@ -8,7 +8,7 @@ Cloud
 
 EdgeDB Cloud is the easiest way to host your EdgeDB instance. We offer two ways
 to interact with EdgeDB Cloud: via our CLI or through a graphical web
-interface.
+interface nearly identical to the :ref:`ref_cli_edgedb_ui`.
```

Comment:
It's a little pedantic, but "the EdgeDB UI" or something like "the interface summoned by the edgedb ui command" so it doesn't look like we're playing fast and loose with our branding. Should be capitalized unless we're referring to the command, in other words.
```suggestion
interface nearly identical to the :ref:`EdgeDB UI <ref_cli_edgedb_ui>`.
```